datastore/tests/unit/test_transaction.py

```diff
@@ -15,7 +15,30 @@
 import unittest
 
 import mock
+from google.cloud.datastore.transaction import TransactionOptions
```

Comment:
We follow a rule in our **unit** tests (but not system) that imports of the code being tested happen at run time, not import time. So you'd move this import into `_get_target_class()` so that the code only gets imported when the test is run.

Reply:
ok.